libs/designer/src/lib/core/utils/__test__/segment.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { beforeAll, describe, expect, it } from 'vitest';
+import { createLiteralValueSegment, ValueSegmentConvertor, ValueSegmentConvertorOptions } from '../parameters/segment';
+import constants from '../../../common/constants';
+
+describe('convertToValueSegments', () => {
+  const convertor = new ValueSegmentConvertor({
+    shouldUncast: true,
+    rawModeEnabled: true,
+  });
+  it('properly converts string segment to Value Segment', () => {
+    const stringValue = 'test';
+    const result = convertor.convertToValueSegments(stringValue, constants.SWAGGER.TYPE.STRING);
+    expect(result[0].value).toEqual(createLiteralValueSegment(stringValue).value);
+  });
+
+  it('properly wraps string boolean with quotes if of type any', () => {
+    const stringValue = 'true';
```

Comment:
What happens when we want to input number or boolean will it always be enclosed in quotes?